src/pages/Environments.js

```diff
@@ -456,8 +456,8 @@ const Environments = () => {
   // created, workspace namespace (a.k.a billing project) value used to equal the google project.
   // Therefore we use google project if the namespace label is not defined.
   const renderWorkspaceForApps = app => {
-    const { appType, googleProject, labels: { saturnWorkspaceNamespace = googleProject, saturnWorkspaceName } } = app
-    return getWorkspaceCell(saturnWorkspaceNamespace, saturnWorkspaceName, appType, false)
+    const { appType, cloudContext: { cloudResource }, labels: { saturnWorkspaceName } } = app
+    return getWorkspaceCell(cloudResource, saturnWorkspaceName, appType, false)
```

Comment:
I think we should still be using `saturnWorkspaceNamespace` if it exists...and only fallback to `cloudResource` if the label doesn't exist